### ISSUE TITLE
Update license text in metadata file?

### ISSUE THREAD

**Issue by klauswyser** [EC-Earth/ece2cmor3#738]
The license of CMIP6 models including EC-Earth3 will be changed to CC BY 4.0 (removing "Shared Alike"), see [1]. Should this change also be reflected in the metadata template in ece2cmor3? Or maybe this is not necessary any longer because license handling will be centralised according to an e-mail from Paul Dirac:

> Once updated, the revised licenses will propagate across all related CMIP6 resources (changes only need to be made in the CMIP6_CVs). These will be visible on the DOI landing pages, in the ESGF portals, and on the licenses page (linked above), which we will also link off the CMIP6 Guide page (https://pcmdi.llnl.gov/CMIP6/Guide/dataUsers.html). The centralization of the current license information ensures that the latest license information is collated, transparent, and always available for the entire archive.

[1] https://dev.ec-earth.org/projects/cmip6/wiki/License_on_EC-Earth3_CMIP6_data#Update-03-06-2022

**Comment by treerink** [comment]
As far I understand they will 'fix' this at the CMIP6 citation pages (DOI landing pages). Adjusting the  metadata template of course will make the change of the licence in the netcdf meta data for future cmorisation. For published data on the ESGF nodes a cmor-fixer option could be added, but I think republishing all current published EC-Earth data is too much effort (there is the waiver). A last point is that there are also non CMIP projects using ece2cmor3, if I change the metadata template the license will be changed (without the SA), nobody will notice of course which is a little tricky. Add the other hand most won't care I guess.

**Comment by klauswyser** [comment]
I agree that it's not worth the effort to fix the license in all existing ECE3 data but I still think it would be nice to update the metadata template that is distributed with the ece2cmor3 tool so all future publications become correct.

Other than CMIP6  datasets are of no concern here, such experiments/variables will have different metadata anyway, including a different license.

**Comment by treerink** [comment]
Okay, let's update that from our side. They should update this site:
 https://pcmdi.llnl.gov/CMIP6/TermsOfUse/TermsOfUse6-1.html
because the netcdf metadata refers to this one. The following files will be updated:
```
resources/metadata-templates/cmip6-CMIP-piControl-metadata-template.json 
scripts/data-qa/recipes/sftlf-AOGCM.sh                                     # Only needed for version < 1.8.0
```

**Comment by treerink** [comment]
Done.
